packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2679,6 +2680,11 @@ export interface LocalVideoStreamVideoEffectsState {
     isActive: boolean;
 }
 
+// @beta
+export interface LocalVideoTileOptions {
+    position?: 'grid' | 'floating' | 'hidden';
+}
+
 // @beta
 export type LocalVideoTileSize = '9:16' | '16:9' | 'hidden' | 'matchStream';
```

Comment:
`hidden` is now in two places, can we remove hidden from here now?

Reply:
never mind the new property is just for composites

Reply:
we still need to pass it into the local tile options in the component form the composite. I am not sure how we could pull it out.